divi/src/masternode-payments.cpp

```diff
@@ -19,10 +19,13 @@
 #include <boost/lexical_cast.hpp>
 #include <algorithm>
 #include <numeric>
-
 #include <SuperblockHelpers.h>
 #include <LotteryWinnersCalculator.h>
 #include <BlockIncentivesPopulator.h>
+#ifndef BITCOIN_MAIN_H
+#include <main.h>
+#endif // BITCOIN_MAIN_H
+
```

Comment:
As mentioned previously regarding double header guards

Reply:
fixed it